scss/components/_icons.scss

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file
+ * Contains class rules for material icons with different font sizes
+ *
+ * Usage
+ * To add a icon to the HTML document add a <i> tag with class `material-icons`
+ * e.g. <h2> <i class="material-icons">face</i> Title </h2>
+ * To know better refer - https://google.github.io/material-design-icons/#using-the-icons-in-html
+ * To see the list of all available icons - https://material.io/tools/icons/?style=baseline
+ */
+
+.material-icons {
```

Comment:
The components folder is for the CiviCase components like `case-card`, `activity-card` etc. Also `material-icons` is not exactly a component in my opinion. Its just a helper class. So this should be part of `_material-icons.scss`.